src/functions/searches/initiate-search.ts

```diff
@@ -49,9 +50,13 @@ class QueryQueue {
 		};
 
 		// Check query tasks once that's completed
-		taskPromise.then(() => {
-			this._checkQueryTasks(query);
-		});
+		taskPromise
+			.then(() => {
+				this._checkQueryTasks(query);
+			})
+			.catch(() => {
+				return;
```

Comment:
I found that without this `catch()`, the test runner would always error with `Unhandled promise rejection`. This assertion...

https://github.com/gravwell/js-client/blob/c90184f012ce93685cc67036c8919a5120f60214/src/functions/searches/subscribe-to-one-search/subscribe-to-one-search.spec.ts#L381

... would work as expected, and the test would pass. However, after the test completed I would inevitably get an unhandled promise rejection error.

![errors](https://user-images.githubusercontent.com/154231/112075210-ac9a6e80-8b3d-11eb-823f-5686a723177a.png)